### PR TITLE
Add docker scripts

### PR DESCRIPTION
**Pull request by l4l**
Add:
- Building local image
- Building test swarm

Besides add absolute path to swarm-deploy due to some issues

### CODE REVIEW

File: scripts/build-local-image.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+PROJ_BIN=/opt/iroha
+docker pull hyperledger/iroha-docker-develop
+# docker build $IROHA_HOME/docker/develop -t iroha-dev
+echo $IROHA_HOME\;$MOUNT\;$BUILD_PATH\;$PROJ_BIN
+# docker run -tdv $IROHA_HOME:$MOUNT hyperledger/iroha-docker-develop "/bin/bash" >/tmp/iroha-build.id
+CONTAINER=$(cat /tmp/iroha-build.id)
+docker exec $CONTAINER bash -c "mkdir -p $BUILD_PATH $PROJ_BIN;
+	cd $BUILD_PATH && cmake -DTESTING=OFF $MOUNT && make;
+	cp $BUILD_PATH/bin/* $PROJ_BIN;
+	ldd $PROJ_BIN/* | egrep -o '\/.*?\.(so(\..*?)?|a)\s' | grep $MOUNT | uniq | xargs cp -t $PROJ_BIN | true;"
```
Comment (Warchant) [inline scripts/build-local-image.sh]: Change to 

```bash
docker exec $CONTAINER sh << COMMANDS
# bash commands here
COMMANDS
```

As [here] (https://github.com/hyperledger/iroha/blob/master/docker/build_in_docker.sh#L22-L51)

File: scripts/build-local-image.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (Warchant) [inline scripts/build-local-image.sh]: change to 
```
#!/bin/sh -x
```

File: scripts/build-test-swarm.sh [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
```
Comment (Warchant) [inline scripts/build-test-swarm.sh]: sh -x

File: scripts/build-test-swarm.sh [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+if [ -z "$1" ] || [ "$1" -le "0" ]; then
+    echo "Usage: $0 <number_of_peers>"
+    exit -1
+fi
+PEERS_NUM=$($1-1 | bc)
```
Comment (Warchant) [inline scripts/build-test-swarm.sh]: Explain this please. 

On my PC this does not work.

File: scripts/swarm-deploy.sh [hunk 5]
```diff
@@ -69,9 +69,9 @@ do
     # copy config
     docker cp ${IROHA_HOME}/iroha.conf ${COMPOSE_PROJECT_NAME}_node_1:/
     # generate genesis block based on list of peers
-    docker exec ${COMPOSE_PROJECT_NAME}_node_1 iroha-cli --genesis_block --peers_address /peers.list
+    docker exec ${COMPOSE_PROJECT_NAME}_node_1 /opt/iroha/iroha-cli --genesis_block --peers_address /peers.list
     # run irohad with output redirection to /iroha.log
-    docker exec -d ${COMPOSE_PROJECT_NAME}_node_1 bash -c "irohad --config /iroha.conf --genesis_block /genesis.block --peer_number ${i} > /iroha.log"
+    docker exec -d ${COMPOSE_PROJECT_NAME}_node_1 bash -c "/opt/iroha/irohad --config /iroha.conf --genesis_block /genesis.block --peer_number ${i} > /iroha.log"
```
Comment (Warchant) [inline scripts/swarm-deploy.sh]: why `${SMTH}_node_1? 
All irohad services will  be connected to the same postgres and redis...

Comment (Warchant) [reply scripts/swarm-deploy.sh]: Should be 

```bash
${COMPOSE_PROJECT_NAME}_node_$number
```

**Comment by Warchant** [comment]
Consider using `Ansible` for deploy automation.

File: scripts/build-test-swarm.sh (hunk 4 above)
Comment (l4l) [reply scripts/build-test-swarm.sh]: My mistake, yeah. Just wanted to decrement value via bc, e.g `echo 1-1 | bc` gives 0. I'll replace with `$((..))`

File: scripts/swarm-deploy.sh (hunk 5 above)
Comment (l4l) [reply scripts/swarm-deploy.sh]: suffix means here replica number and for now it's just 1
ID of node is in the  `$COMPOSE_PROJECT_NAME`

**Pull request merged**